Source/App/EbAppMain.c

```diff
@@ -184,13 +190,15 @@ int main(int argc, char* argv[])
                         return_error = (EB_ERRORTYPE)(return_error & return_errors[instanceCount]);
                         exitConditions[instanceCount]       = APP_ExitConditionNone;
                         exitConditionsOutput[instanceCount] = APP_ExitConditionNone;
+                        exitConditionsRecon[instanceCount]  = configs[instanceCount]->reconFile ? APP_ExitConditionNone : APP_ExitConditionError;
```

Comment:
init exitConditionsRecon to APP_ExitConditionNone; can also avoid if() problem below.